dev/how-to/get-started/data-migration.md

```diff
@@ -0,0 +1,649 @@
+---
```

Comment:
Please add this file to TOC.md.

Reply:
@lilin90 Added. https://github.com/pingcap/docs/pull/1157